ecus.md

```diff
@@ -7,12 +7,12 @@ css_id: ecus
 
 At the highest level, the basic requirement for an ECU to be capable of supporting Uptane is that it must be able to perform either full or partial verification, and have a secure source of time. (See the [Uptane Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#build-time-prerequisite-requirements-for-ecus) for official requirements.)
 
-To bootstrap an Uptane-capable ECU, a few things need to be provisioned into the ECU:
+To bootstrap an Uptane-capable ECU, a few things need to be provisioned into unit:
```

Comment:
"it"?

Reply:
yes. Thanks for the catch.

Reply:
I'll check on the graphic. It was loaded to the repo, but perhaps I did not properly enter the image name.